GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/java/org/gearvrf/io/cursor3d/Cursor.java

```diff
@@ -461,8 +461,46 @@ public IoDevice getIoDevice() {
     }
 
     /**
-     * Returns a list of {@link IoDevice}s compatible with the {@link Cursor}.
+     * Returns the priority for the {@link IoDevice} currently attached to the {@link Cursor}.
+     *
+     * @return -1 if no device attached, otherwise returns device priority
+     */
+    public int getCurrentDevicePriority() {
```

Comment:
Should getCurrentDevicePriority, getDevicePriority, isDeviceCompatible be public? What use can they be to end users?

Reply:
I agree, these are only used by the cursor manager.